### PR TITLE
Use apidoc directive in typed/cluster.md (#22904)

### PR DESCRIPTION
**Pull request by Captain1653**
References #22904

### CODE REVIEW

**Review by ennru — APPROVED** [review]
LGTM.

**Comment by Captain1653** [comment]
Can it be reviewed in 2.6.19? Please :)

File: akka-docs/src/main/paradox/typed/cluster.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ The Cluster extension gives you access to management tasks such as @ref:[Joining
 and subscription of cluster membership events such as @ref:[MemberUp, MemberRemoved and UnreachableMember](cluster-membership.md#membership-lifecycle),
 which are exposed as event APIs.  
 
-It does this through these references on the `Cluster` extension:
+It does this through these references on the @apidoc[typed.Cluster] extension:
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/cluster.md:45]: Perhaps:
```suggestion
It does this through these references on the @apidoc[typed.Cluster$] extension:
```

File: akka-docs/src/main/paradox/typed/cluster.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ The Cluster extension gives you access to management tasks such as @ref:[Joining
...
 
-* `manager`: An @scala[`ActorRef[akka.cluster.typed.ClusterCommand]`]@java[`ActorRef<akka.cluster.typed.ClusterCommand>`] where a `ClusterCommand` is a command such as: `Join`, `Leave` and `Down`
-* `subscriptions`: An @scala[`ActorRef[akka.cluster.typed.ClusterStateSubscription]`]@java[`ActorRef<akka.cluster.typed.ClusterStateSubscription>`] where a `ClusterStateSubscription` is one of `GetCurrentState` or `Subscribe` and `Unsubscribe` to cluster events like `MemberRemoved`
-* `state`: The current `CurrentClusterState`
+* `manager`: An @scala[@apidoc[typed.ActorRef]\[@apidoc[akka.cluster.typed.ClusterCommand](typed.ClusterCommand)\]]@java[@apidoc[typed.ActorRef]<@apidoc[akka.cluster.typed.ClusterCommand](typed.ClusterCommand)>] where a `ClusterCommand` is a command such as: @apidoc[Join], @apidoc[Leave] and @apidoc[Down]
```
Comment (raboof) [inline akka-docs/src/main/paradox/typed/cluster.md:47]: nice

**Review by raboof — APPROVED** [review]

**Pull request merged**